### PR TITLE
Admin vote page: new look

### PR DESCRIPTION
**Pull request by davidwessman**
Adding more disposition and current vote status
Adding refresh action for amount of votes

### CODE REVIEW

File: app/controllers/admin/votes_controller.rb [hunk 1]
```diff
@@ -66,14 +66,16 @@ def open
       flash[:alert] = vote.errors[:status].to_sentence
     end
 
-    redirect_to admin_votes_path
+    redirect_to redirect_path(vote, params[:route])
```
Comment (navro) [inline app/controllers/admin/votes_controller.rb]: Nice solution to this problem :)

File: app/views/admin/votes/edit.html.erb [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <% provide(:title, t('vote.admin_votes')) %>
 <div class="col-md-6 col-md-offset-3 col-sm-12 reg-page">
   <div class="headline">
-    <h2><%= t('vote.edit_vote') %></h2>
+    <h1><%= t('vote.edit_vote') %></h12>
```
Comment (navro) [inline app/views/admin/votes/edit.html.erb]: h12? :wink:

Comment (davidwessman) [inline app/views/admin/votes/edit.html.erb]: Slippery keys!

**Comment by davidwessman** [comment]
@navro How's this one going?

**Comment by navro** [comment]
> @navro How's this one going?

I thought you were going to finish it...

**Comment by davidwessman** [comment]
Need to fix `@vote` and `@vote_status.vote` 😄

**Comment by davidwessman** [comment]
@navro - please check now 😄

**Comment by navro** [comment]
Looks good! :+1:
+1

**Pull request merged**